nucypher/cli/processes.py

```diff
@@ -56,9 +56,6 @@ def __init__(self, ursula, emitter):
             'known_nodes': self.paintKnownNodes,
             'fleet_state': self.paintFleetState,
 
-            # Blockchain Control
-            'commit_next': self.commit_to_next_period,  # hidden
-
```

Comment:
Another mention of this command on line 82

Reply:
Ah - missed it! :heavy_check_mark: